packages/frontend/src/components/SimpleStatistic/index.tsx

```diff
@@ -102,6 +103,15 @@ const SimpleStatistic: FC<SimpleStatisticsProps> = ({
         };
     }, [observerElementSize]);
 
+    const comparisonValueColor = useMemo(() => {
```

Comment:
Is this necessary if both cases return the same colour?

Reply:
it's extended in the next PR 🙈